### PR TITLE
GEOMESA-789 Make SCM and distribution-management properties

### PR DESCRIPTION
**Pull request by cne1x**
Reviewer/committer:  Please also apply this change to the RC-6 release.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,11 +65,21 @@
...
+        <dist.snapshots.name>GeoMesa Repository - Snapshots</dist.snapshots.name>
+        <dist.snapshots.url>https://repo.locationtech.org/content/repositories/geomesa-snapshots/</dist.snapshots.url>
         <dist.accumulo.lib.ext>accumulo/lib/ext</dist.accumulo.lib.ext>
         <dist.hdfs>hdfs</dist.hdfs>
         <dist.geoserver.lib>geoserver/WEB-INF/lib</dist.geoserver.lib>
         <dist.dev>dev</dist.dev>
 
+        <!-- SCM properties -->
+        <scm.connection>scm:git:git@gitlab.devforce.io:tcri-geospatial/tcri-geospatial-geoserver-authorizations-provider.git</scm.connection>
+        <scm.url>https://gitlab.devforce.io/tcri-geospatial/tcri-geospatial-geoserver-authorizations-provider.git</scm.url>
```
Comment (elahrvivaz) [inline pom.xml]: this should point to locationtech, right?

**Comment by anthonyccri** [comment]
Can we use the -DaltDeploymentRepository instead of this?

**Comment by cne1x** [comment]
@anthonyccri I assume that we could, but I admit that ```-DaltDeploymentRepository``` feels clunky to me, even more than the proposed solution here (for what little that's worth), and it leaves the SCM hard-wired.  (We could certainly use ```-Dproject.scm.developerConnection``` and its ilk for the SCM on the command-line, too.)  I think the current PR improves readability for folks who want their own private GeoMesa repo/SCM, but I'm easy either way.  Thanks!

**Comment by anthonyccri** [comment]
Seems like ```-DaltDeploymentRepository``` is the standard way to do this sort of thing ... I don't think I've ever seen a project use properties in SCM settings.

**Comment by cne1x** [comment]
Withdrawn as per discussion, above.

**Pull request closed without merging**